src/App.tsx

```diff
@@ -1,18 +1,37 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const onPerPageChange = (event: ChangeEvent<HTMLSelectElement>) => {
```

Comment:
```suggestion
  const handlePerPageChange = (event: ChangeEvent<HTMLSelectElement>) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
thanks!